src/muqsit/chestshop/Category.php

```diff
@@ -87,11 +82,14 @@ public function send(Player $player, int $page = 1, bool $send = true) : int{
 
 			$page = min(ceil(count($this->items) / 45), max(1, $page));//page > 0 and page <= number of pages.
 			$contents = array_slice($this->items, ($page - 1) * 45, 45);//get the 45 items on this page.
+
+			$contents[48] = Button::get(Button::TURN_LEFT, $this->getRealName());
+			$contents[49] = Button::get(Button::CATEGORIES);
+			$contents[50] = Button::get(Button::TURN_RIGHT, $this->getRealName());
+
 			$inventory = $this->menu->getInventory($player);
 
-			foreach($contents as $slot => $item){
-				$inventory->setItem($slot, $item, false);
-			}
+			$inventory->setContents($contents);
```

Comment:
Now that that's the only method being called from `$inventory`, you might aswell directly call it as
`$this->menu->getInventory($player)->setContents($contents);`. The variable declaration becomes uneeded.